dashboard/app/controllers/projects_controller.rb

```diff
@@ -15,4 +21,16 @@ def template
 
     render template: "projects/#{params[:template]}", layout: nil
   end
+
+  def new
```

Comment:
I would maybe make is_project a property of the level (and maybe rename it to is_standalone since we're going to have projects in freeplay levels).

And I think if you do it that way you don't even need the instance variable since you only use it from the json-ized level